public/app/features/live/pages/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { getBackendSrv } from '@grafana/runtime';
+import { PipelineListOption, EntitiesTypes, PipeLineEntitiesInfo } from './types';
+
+export async function getPipeLineEntities(): Promise<PipeLineEntitiesInfo> {
+  return await getBackendSrv()
+    .get(`api/live/pipeline-entities`)
+    .then((data) => {
```

Comment:
not needed now -- but this is a good candidate for adding a jest test.  We may also want to make it singleton so we do not need to pass it around as props.